jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerConfigCredentialRetriever.java

```diff
@@ -89,52 +77,45 @@ public Credential retrieve() throws IOException {
     if (dockerConfigTemplate == null) {
       return null;
     }
-
-    DockerConfig dockerConfig = new DockerConfig(dockerConfigTemplate);
-
-    for (String registryAlias : RegistryAliasGroup.getAliasesGroup(registry)) {
-      Credential credentials = retrieve(dockerConfig, registryAlias);
-      if (credentials != null) {
-        return credentials;
-      }
-    }
-    return null;
+    return retrieve(new DockerConfig(dockerConfigTemplate));
   }
 
   /**
    * Retrieves credentials for a registry alias from a {@link DockerConfig}.
    *
    * @param dockerConfig the {@link DockerConfig} to retrieve from
-   * @param registryAlias the registry alias to use
    * @return the retrieved credentials, or {@code null} if none are found
    */
+  @VisibleForTesting
```

Comment:
This seems to change the tests to skip the logic in the main `retrieve()` method, like loading the test Docker config? Was there a reason for testing this private method instead?

Reply:
That logic is still covered in the `hasAuth` and `nonexistentDockerConfigFile` tests; this change was so I could mock `dockerConfig#getCredentialHelperFor` without using a factory.